sunpycube/cube/NDCube.py

```diff
@@ -132,6 +133,17 @@ def __getitem__(self, item):
         return result
 
 
+class NDCubeOrdered(NDCube):
```

Comment:
This has to alter both the shape of the array and the wcs.  You can take out code from orient() in cube_utils that does this.